--- qemacs-0.3.1/charset.c~	2003-04-22 00:01:42.000000000 +0200
+++ qemacs-0.3.1/charset.c	2005-11-03 16:22:51.000000000 +0100
@@ -20,7 +20,7 @@
 
 QECharset *first_charset = NULL;
 
-extern QECharset charset_7bit;
+static QECharset charset_7bit;
 
 /* specific tables */
 static unsigned short table_idem[256];
--- qemacs-0.3.1/tty.c~	2003-04-22 00:01:42.000000000 +0200
+++ qemacs-0.3.1/tty.c	2005-11-03 16:26:30.000000000 +0100
@@ -70,7 +70,7 @@
     return 1;
 }
 
-extern QEDisplay tty_dpy;
+static QEDisplay tty_dpy;
 
 static int term_init(QEditScreen *s, int w, int h)
 {
--- qemacs-0.3.1/x11.c~	2003-04-22 00:01:42.000000000 +0200
+++ qemacs-0.3.1/x11.c	2005-11-03 16:28:00.000000000 +0100
@@ -76,7 +76,7 @@
 static CSSRect update_rects[UPDATE_MAX_REGIONS];
 #endif
 
-extern QEDisplay x11_dpy;
+static QEDisplay x11_dpy;
 static int visual_depth;
 
 static int force_tty = 0;
